frontend/actions/animeList.js

```diff
@@ -23,24 +23,26 @@ export function getSeason(year: number, season: string) {
       return false
 
     if (token)
-      getAnimeSeason(year, season, token.access_token)
+      getAnimeSeason({year, season}, token.access_token)
     else
       dispatch(getAccessToken())
-        .then(newToken => getAnimeSeason(year, season, newToken.access_token))
+        .then(newToken => getAnimeSeason({year, season}, newToken.access_token))
 
-    function getAnimeSeason(year: number, season: string, accessToken: string) {
+    function getAnimeSeason({year: number, season: string}, accessToken: string) {
       dispatch(loadingSeasonPending())
-      return AL.getAnimeSeason(year, season, accessToken)
+      return AL.getAnimeSeason({year, season}, accessToken)
          .then(data => {
-           dispatch(setSeason(year, season, data))
+           dispatch(setSeason({year, season}, data))
            dispatch(loadingSeasonComplete())
          })
          .catch(err => dispatch(serverError(err)))
     }
   }
 }
 
-export function setSeason(year: number, season: string, animes: Array<Object>) {
+export function setSeason(filter:Object, animes: Array<Object>) {
```

Comment:
You can destructure the filter object here too so you don't have to directly pull them out like you're doing.